src/web/components/form/radio.jsx

```diff
@@ -30,7 +31,7 @@ const Radio = ({
       {...props}
       checked={checked}
       disabled={disabled}
-      label={String(title)}
+      label={isString(title) && title}
```

Comment:
Maybe we could check if title has a `toString` method? Something like this?
```suggestion
      label={title?.toString && String(title)}
```

Reply:
We could encounter issues with `booleans` and `truthy` values. If we want the component to convert it just in case then we will need.

```js
  const getLabel = title => {
    if (typeof title === 'string' || typeof title === 'number') {
      return String(title);
    }
    return null;
  };
```

Reply:
sadly this wont work with the `_l` lazy translation function :-( so we should be really careful what to expect.